.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           fetch-depth: 0
 
       - name: Setup Java and Scala
-        uses: olafurpg/setup-scala@v12
+        uses: olafurpg/setup-scala@v10
```

Comment:
Why did this get incremented backwards?

Reply:
Did I mess up the circular dependency with `sbt-spiewak`?

Reply:
Maybe! I didn't notice the update of that dependency. I have an idea.